src/pydash/functions.py

```diff
@@ -180,6 +188,14 @@ def __call__(self, *args: P.args, **kwargs: P.kwargs) -> T:
         # type safety is ensured from the `__init__` signature
         return result  # type: ignore
 
+    @property
+    def _from_index(self) -> int:
+        return -1 if self.from_right else 0
```

Comment:
Yeah sounds good. I left it as `property` in case `from_right` was changed but if `_from_index` is marked starting with a underscore and the user changes it I guess it's on him if it doesn't work properly anymore